### DISCUSSION TITLE
CloudWatch: Display notice on timeseries panel for ArithmeticError instead of breaking the render

### DISCUSSION THREAD

**Discussion by fridgepoet** [grafana/grafana#45760] [Dashboards, Panels & Visualizations]
### Display softer warning/notice for ArithmeticError and render frames

There is a query that returns and error *and* frames. 
Proposed feature request: Display warning and frames

Current behavior: Only the error is rendered
<img src ="https://user-images.githubusercontent.com/4163034/155284801-ac636b40-dd4e-436d-8291-bfc0732a996a.png" width="600">

On AWS console they display a little warning:
<img src ="https://user-images.githubusercontent.com/4163034/155284847-7d3c3c1f-d8a5-4cf3-848d-0972fcdaea3d.png" width="600">

[Related support escalation](https://github.com/grafana/support-escalations/issues/1860)

**Comment by fridgepoet** [comment]
From @sunker :

I think one path forward would be to return a notice instead of an error. That way a warning icon will be displayed at the top of the panel, and if you hover over it you'll see the actual message.
```
frame.AppendNotices(data.Notice{
	Severity: data.NoticeSeverityWarning,
	Text:     "the message",
})
```

https://github.com/grafana/grafana/blob/main/pkg/tsdb/cloudwatch/response_parser.go#L193-L198

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
